actors/miner/src/lib.rs

```diff
@@ -4587,9 +4592,9 @@ fn handle_proving_deadline(
         Ok(state.clone())
     })?;
 
-    // Remove power for new faults, and burn penalties.
+    // Remove power for new faults, and burn penalties and fees.
     request_update_power(rt, power_delta_total)?;
-    burn_funds(rt, penalty_total)?;
+    burn_funds(rt, burn_total)?;
```

Comment:
I'm thinking there's a fee debt case that's not handled here

should the fee perhaps be done through `apply_penalty` first so that the burn comes out of `repay_partial_debt_in_priority_order` properly and then remains in `fee_debt` if there's no balance?

Reply:
I don't quite understand what you're doing but the intended usage is 
1) apply_penalty for all fee payments adding to the fee debt
2) pay off fees in the following order
  a) locked funds
  b) available balance 
  c) remaining debt to be paid later

Since I'm pretty sure we want to maintain this payment ordering I think what you want to do is add the recurring payment through apply_penalty and keep a single call to repay_debts_in_priority_order to pay off everything.